src/track/beatstats.cpp

```diff
@@ -0,0 +1,84 @@
+#include <QList>
```

Comment:
There is also another class that is not windowed in this file. Also like I mention somewhere in another comment that we use a double as the key it`s "tuned" to work only on the bpm values and may fail elsewhere.
bpmStatics would be more truthful, but, beatStastics will keep it close to the beatutils where it belongs...

Reply:
Normally we use a file per class. So finally it would be the best to spit the file up and name it according to the class. 
The statistic functions have nothing to do with beats. The calling context defines what is in the values. So I think we should keep the names neutral.